README.md

```diff
@@ -17,6 +17,38 @@
 - [TLS証明書作成](/doc/setup-of-ACM.md)
 - [ロードバランサー構築](/doc/setup-of-ALB.md)
 - [RDS構築](/doc/setup-of-RDS.md)  
+- [ECR構築](/doc/setup-of-ECR.md)
+- [EC2構築](/doc/setup-of-EC2.md)  
+
+## dockerコンテナのデプロイ  
+前提:  
+・EC2構築済み  
+・ECR構築済み
+
+- プロジェクトディレクトリに移動し、以下のコマンドを実行 
+```
+#本番環境
+docker-compose -f docker-compose-prod.yml up -d   
+
+#検証環境
+docker-compose -f docker-compose-prod.yml up -d   
```

Comment:
こちらでもdocker-compose-prod.ymlを使うのでしょうか？

Reply:
またこちらのコマンドをファイルをdevに変更して実行すると、下記エラーメッセージが表示されるのですが、追加の設定は必要ないのでしょうか？

```
WARNING: The DB_ROOT_PASSWORD variable is not set. Defaulting to a blank string.
WARNING: The DB_NAME variable is not set. Defaulting to a blank string.
WARNING: The DB_USER variable is not set. Defaulting to a blank string.
WARNING: The DB_PASSWORD variable is not set. Defaulting to a blank string.
WARNING: The TIME_ZONE variable is not set. Defaulting to a blank string.
```

Reply:
動作させようとすると他の問題も出てきたので、先にコメントしておきますね。

appコンテナに入って、マイグレーションを実行しようとすると下記エラーになったのですけど、こちらもどのように対処すれば良いでしょうか？

```
root@87c331175862:/var/www/dailyReport# php artisan migrate
**************************************
*     Application In Production!     *
**************************************

 Do you really wish to run this command? (yes/no) [no]:
 > yes


   Illuminate\Database\QueryException

  SQLSTATE[HY000] [2002] Connection refused (SQL: select * from information_schema.tables where table_schema = forge and table_name = migrations and table_type = 'BASE TABLE')

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:712
    708▕         // If an exception occurs when attempting to run a query, we'll format the error
    709▕         // message to include the bindings with SQL, which will make this exception a
    710▕         // lot more helpful to the developer instead of just the database's errors.
    711▕         catch (Exception $e) {
  ➜ 712▕             throw new QueryException(
    713▕                 $query, $this->prepareBindings($bindings), $e
    714▕             );
    715▕         }
    716▕     }

      +36 vendor frames
  37  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

Reply:
ありがとうございます。

> こちらでもdocker-compose-prod.ymlを使うのでしょうか？
こちら誤りで、正しくはdocker-compose-dev.ymlを利用します。
（修正しました。）

> またこちらのコマンドをファイルをdevに変更して実行すると、下記エラーメッセージが表示されるのですが、環境変数周りの追加の設定は必要ないのでしょうか？

Docker用の.envファイルを作成する手順を記載していなかったので、本番開発それぞれ用を作成する初期設定を記載しました。

> appコンテナに入って、マイグレーションを実行しようとすると下記エラーになったのですけど、こちらもどのように対処すれば良いでしょうか？

こちらLaravelソースディレクトリに.envを作成&DB接続情報記載をする初期設定を追記しました。